src/lib/hooks/use-trade.ts

```diff
@@ -71,16 +72,14 @@ export const useTrade = () => {
         // If the user overrides, we take any gas estimate
         const canFail = override
         const gasLimit = await gasEstimatooor.estimate(tx, canFail)
-        tx.gasLimit = BigNumber.from(gasLimit.toString())
-        const request = await prepareSendTransaction({
+        const hash = await walletClient.sendTransaction({
```

Comment:
Preferring to just use `viem` here now because it's way simpler than those hooks for what we need.

Reply:
https://viem.sh/zksync/actions/sendTransaction#sendtransaction